numpy/core/tests/test_array_coercion.py

```diff
@@ -40,8 +40,8 @@ def subclass(a):
 
     class _SequenceLike():
         # We are giving a warning that array-like's were also expected to be
-        # sequence-like in `np.array([array_like])`, this can be removed
-        # when the deprecation exired (started NumPy 1.20)
+        # sequence-like in `np.array([array_like])`. This can be removed
+        # when the deprecation expired (started NumPy 1.20).
```

Comment:
I wonder if it has been expired, or should be. @seberg Thoughts?

Reply:
Should be "expires", I think. But hopefully we can remove the whole thing.

Reply:
It looks like it already is, this isn't used for any warning tests anymore.